src/main/java/bsh/Interpreter.java

```diff
@@ -291,10 +292,10 @@ public Interpreter()
     public void setConsole( ConsoleInterface console ) {
         this.console = console;
         setu( "bsh.console", console );
-        // redundant with constructor
-        setOut( console.getOut() );
-        setErr( console.getErr() );
-        // need to set the input stream - reinit the parser?
```

Comment:
I removed the comment, but maybe we should keep it? the question is valid, we probably should, otherwise there is no point in setting a new Console.